test/io/serializers/arrays.default.good

```diff
@@ -363,11 +363,11 @@ SUCCESS: [domain(4,int(64),one)] int(64)
 SUCCESS: [domain(5,int(64),one)] int(64)
 --- array of arrays ---
 ===== writing: =====
-1 2 3 2 4 6 3 6 9
+1.11235 1.21235 1.31235 2.11235 2.21235 2.31235 3.11235 3.21235 3.31235
 --------------------
-1 2 3 2 4 6 3 6 9
+1.11235 1.21235 1.31235 2.11235 2.21235 2.31235 3.11235 3.21235 3.31235
```

Comment:
I'm not completely convinced these will be reliable, unfortunately :(

Reply:
in terms of floating point operations having different behavior on different systems/compiler-optimizations?

If so, I don't think I'd be too worried about it here since I'm doing some pretty simple mathematical operations and only showing the first 5 digits of precision. Let me know if you think otherwise

Reply:
It was my understanding that floating point representations are imprecise generally, but that concern is more founded on what I've picked up over the years rather than being an expert on the implementation of floating points.  So if we get sporadic failures due to solar flares, I'll say I told you so but I'm okay with yielding :)